db/migrate/20210527114834_set_tagging_enabled.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class SetTaggingEnabled < ActiveRecord::Migration[6.1]
+  def up
+    result = execute('SELECT COUNT(*) FROM topics')
+
+    # keep tagging disabled for existent sites
+    if result.first['count'] > 0
```

Comment:
In theory this should be fine because migrations should be run before seed data, but I just want to confirm you tested this on an empty/pre-existing database manually and it worked?

Another approach that might work is to check the hidden setting `bypass_wizard_check`. It should be set once the wizard is complete for any forum.

Reply:
> In theory this should be fine because migrations should be run before seed data, but I just want to confirm you tested this on an empty/pre-existing database manually and it worked?

Yes, it worked. However, @SamSaffron suggested to use migration data so I am switching to that anyway.